src/mixin/SymbolCheck.js

```diff
@@ -190,5 +183,25 @@ Ext.define('GeoExt.util.Symbol', {
             checkedCache[symbolStr] = isDefined;
             return isDefined;
         }
+        // </debug>
+    },
+
+    /**
+     * @property {String[]} symbols The symbols to check.
+     */
```

Comment:
I wanted to have these show up in classes that this mixin uses. This leads to this docs:

![image](https://cloud.githubusercontent.com/assets/227934/9680282/055f74e2-52f6-11e5-9de0-eda0a627084d.png)

And then the link can be clicked to the docs in the originating class.